index.d.ts

```diff
@@ -0,0 +1,130 @@
+declare module 'fluture' {
+  interface ResolveFunction<T> {
+    (value: T): void
+  }
+  interface RejectFunction {
+    (error: Error): void
+  }
+
+  interface FunctionReturning<T extends Function> {
+    (...t: any[]): T
+  }
+  abstract class Future<T> {
+    done<T> (fn: (err: Error, value: T) => void): Future<T>
+  }
+
+  function after<T> (duration: number, value: string): Future<T>
```

Comment:
Would it be possible to define a `PositiveInteger` and use that to restrict the input for `duration`?